commands/src/main/java/org/wildfly/extras/creaper/commands/socketbindings/RemoveSocketBinding.java

```diff
@@ -0,0 +1,47 @@
+package org.wildfly.extras.creaper.commands.socketbindings;
+
+import org.wildfly.extras.creaper.core.online.OnlineCommand;
+import org.wildfly.extras.creaper.core.online.OnlineCommandContext;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+
+/**
+ * A command for removing socket binding from default socket binding group if not specified else.
+ * If specified socket-binding does not exist, no operation is performed.
+ */
+public final class RemoveSocketBinding implements OnlineCommand {
+
+    private final String socketBindingName;
+    private String socketBindingGroup;
```

Comment:
This should really be `final` and there should be an extra variable in the `apply` method.